src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,88 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const handleSelectPage = (page: number) => {
+    if (currentPage !== page) {
+      onPageChange(page);
+    }
+  };
+
+  const lastPage = Math.ceil(total / perPage);
+  const arrOfPages = getNumbers(1, lastPage).map(n => n);
```

Comment:
The map function is unnecessary here. getNumbers() function already returns an array so you can directly assign it to arrOfPages like this: const arrOfPages = getNumbers(1, lastPage);